docs/examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations # Required for forward class references, see https://peps.python.org/pep-0484/#forward-references
```

Comment:
IMO, I don't think the comment explaining is needed, it would raise some questions if this is the only place we define them and not for other examples